mifosng-android/src/main/res/layout/fragment_add_savings_account.xml

```diff
@@ -17,7 +17,9 @@
         android:layout_width="fill_parent"
         android:layout_height="fill_parent">
 
-        <LinearLayout style="@style/LinearLayout.Base">
+        <LinearLayout
+            android:id="@+id/linear_layout"
```

Comment:
@NikunjGarg2000  Please change I'd to something like 'll_saving_account' or as suitable.

Reply:
Thank you. I have changed. Please review now.